classes/range/RangePrice.php

```diff
@@ -87,7 +98,9 @@ public function add($autodate = true, $null_values = false)
     /**
      * Get all available price ranges.
      *
-     * @return array Ranges
+     * @param int $id_carrier Carrier identifier
+     *
+     * @return array|false All ranges for this carrier, or false on error
```

Comment:
but if we read `executeS` doc block
```
* @return array|false|mysqli_result|PDOStatement|resource|null
```

Reply:
No opinion about that, depends of the Db driver used no ? I prefer stay simple personally

Reply:
I prefer too, that's why I prefer `array|bool` ^^
@eternoendless What should we do?

Reply:
By analyzing the code, in most circumstances you'll get either an array or false, so let's keep it simple.